packages/react/src/components/Coachmark/PositioningContainer/PositioningContainer.tsx

```diff
@@ -92,34 +92,46 @@ function usePositionState(
   };
 
   const updatePosition = (): void => {
-    const { offsetFromTarget, onPositioned } = props;
+    const { offsetFromTarget, onPositioned, target } = props;
```

Comment:
You probably want to take this `target` from `currentProps` below instead of from props (line 103).